crypto/rand/rand_lcl.h

```diff
@@ -82,6 +82,10 @@
  *                                1.5 * (RAND_DRBG_STRENGTH / 8))
  */
 
+/*
+ * Initial allocation minimum.
+ */
+# define RAND_POOL_MIN_ALLOC     16
```

Comment:
From the (quick) tests I've done, there is a 40 byte use (24 + 16) which causes two expansions in `rand_drbg_get_nonce`.  Neither asks for entropy, so would be allocated in normal memory.

The secure allocations (i.e. those requesting entropy) appear to have the lower bound specified at the exact size required (e.g. 256 bits -> 32 bytes and 384 bits -> 48 bytes) and a large upper bound.  I think that all of the current entropy sources assume a quality of eight bits per byte, which would allow the minimum size to be used as is.  I'd expect some of the digest based DRBGs to go as low as 128 bits.

I'm leaning towards two different minima -- one for secure memory and one for normal: 16 and 40 bytes respectively seem natural.  The secure memory minimum would, ideally, be a power of two due to how the allocator works, the normal arguably shouldn't be but I don't expect it would matter too much.

Reply:
I've set this up to use 16 bytes in secure memory and 48 for normal.
The values are kind of arbitrary but ought to be reasonable for our default setup.